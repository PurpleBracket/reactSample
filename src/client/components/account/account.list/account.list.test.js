/* global describe test expect */
import AccountList from './account.list'
import React from 'react'
import { shallow } from 'enzyme'
import { Provider } from 'react-redux'

describe('Account', () => {
  test('Snapshot', () => {
    const account = shallow(<Provider><AccountList /></Provider>)
    expect(account).toMatchSnapshot()
  })
})
