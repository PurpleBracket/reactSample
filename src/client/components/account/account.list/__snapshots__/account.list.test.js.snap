// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Account Snapshot 1`] = `<Connect(withRouter(AccountList)) />`;
